tools/ui-components/src/colors.css

```diff
@@ -33,37 +33,25 @@
   --red90: #850000;
 }
 
-:export {
-  --gray00: var(--gray00);
-  --gray05: var(--gray05);
-  --gray10: var(--gray10);
-  --gray15: var(--gray15);
-  --gray45: var(--gray45);
-  --gray75: var(--gray75);
-  --gray80: var(--gray80);
-  --gray85: var(--gray85);
-  --gray90: var(--gray90);
-
-  --purple10: var(--purple10);
-  --purple50: var(--purple50);
-  --purple90: var(--purple90);
-
-  --yellow10: var(--yellow10);
-  --yellow20: var(--yellow20);
-  --yellow50: var(--yellow50);
-  --yellow90: var(--yellow90);
-
-  --blue10: var(--blue10);
-  --blue20: var(--blue20);
-  --blue30: var(--blue30);
-  --blue50: var(--blue50);
-  --blue90: var(--blue90);
-
-  --green10: var(--green10);
-  --green90: var(--green90);
+html,
+div.light {
```

Comment:
`div.light` and `div.dark` selectors are added so we could change the wrapper's class conditionally and have the color variable change based on the selected theme.